h2o-core/src/main/java/water/network/CertAndKeyGenWrapper.java

```diff
@@ -0,0 +1,40 @@
+package water.network;
+
+import sun.security.x509.X500Name;
```

Comment:
OpenJDK 👍 

IBM VM - it seems they have a different implementation (see https://www.ibm.com/developerworks/community/forums/html/topic?id=77777777-0000-0000-0000-000013981506) The problem is that this patch will be probably used in IBM environment.  I just would like to make sure it will work. 

Nevertheless, it is not show stopper now.